### ISSUE TITLE
Hide search field

### ISSUE THREAD

**Issue by cklemming** [tamtakoe/oi.select#45]
Possible to ask for a future enhancement to be able to hide the search field and only allow drop-down selection?

Context: [My app](https://waitwhile.com) only wants users to be able to select among the pre-defined options. And since it's just a few options to choose from, the search bar adds unnecessary complexity. Therefore I'd like to hide the search bar - thus make it  behave more like a native select field - but still allow multi-select (i.e. selected options would be added as tags).

**Comment by tamtakoe** [comment]
It is work. You can use `readonly` option

**Comment by cklemming** [comment]
Perfect! Sorry, not sure how I missed that.

**Comment by cklemming** [comment]
Is there a way to get the native Select functionality when you disable search? I'm thinking specifically on mobile, where the native select drop-down is a modal one which is very user-friendly. Attached image below:
![screenshot](https://cloud.githubusercontent.com/assets/10369142/10010129/418c6b8e-609b-11e5-9143-4e3ffe9bd12c.png)

**Comment by tamtakoe** [comment]
Yes. I checked oi.select on mobile device. It is uncomfortable, especially in multiple mode. I think of something, but it's not an easy task.

**Comment by qq645381995** [comment]
You can add the search function? When a lot of data, initialization is only part of the data
